ansible/idr-ms-image-region.yml

```diff
@@ -0,0 +1,23 @@
+- hosts: "{{ idr_environment | default('idr') }}-omeroreadonly-hosts"
+  become: true
+
+  pre_tasks:
+    - name: get database host
+      set_fact:
+        database_server_url: >-
+          {{
+            hostvars[groups[idr_environment | default('idr') + '-database-hosts'][0]]
+            ['ansible_' + (idr_net_iface | default('eth0'))]['ipv4']['address']
+          }}
+    - name: get database password
+      set_fact: >-
+        database_user_password: "{{ idr_secret_postgresql_password_ro | default('omero') }}"
+  roles:
+    - role: ome.omero_ms_image_region
+      omero_ms_image_region_update_nginx: true
```

Comment:
Could these variables be migrated to https://github.com/IDR/deployment/blob/master/ansible/group_vars/omeroreadonly-hosts.yml for consistency and to simplify the maintenance moving forward?